src/components/Dropdown/index.jsx

```
@@ -107,61 +111,84 @@ const Dropdown = ({
+              <MenuItem
+                disabled
+                value={placeholder}
+                sx={{
+                  backgroundColor: 'var(--lightBackground)',
+                  color: 'var(--font)',
+                }}
+              >
+                <p>{placeholder}</p>
+              </MenuItem>
+            )}
+            {listData.map((item, i) => (
```

ESLint

[error] Delete `··`

(prettier/prettier)